core/src/com/unciv/ui/screens/civilopediascreen/CivilopediaCategories.kt

```diff
@@ -21,95 +23,91 @@ enum class CivilopediaCategories (
     ),
     Terrain ("Terrains",
         CivilopediaImageGetters.terrain,
         KeyboardBinding.PediaTerrains,
         "OtherIcons/Terrains",
-        { ruleset, _, _ -> ruleset.terrains.values }
+        { ruleset, _ -> ruleset.terrains.values }
     ),
     Improvement ("Tile Improvements",
         CivilopediaImageGetters.improvement,
         KeyboardBinding.PediaImprovements,
         "OtherIcons/Improvements",
-        { ruleset, _, _ -> ruleset.tileImprovements.values }
+        { ruleset, _ -> ruleset.tileImprovements.values }
     ),
     Unit ("Units",
         CivilopediaImageGetters.construction,
         KeyboardBinding.PediaUnits,
         "OtherIcons/Shield",
-        { ruleset, _, _ -> ruleset.units.values }
+        { ruleset, _ -> ruleset.units.values }
     ),
     UnitType ("Unit types",
         CivilopediaImageGetters.unitType,
         KeyboardBinding.PediaUnitTypes,
         "UnitTypeIcons/UnitTypes",
-        { ruleset, _, _ -> BaseUnitType.getCivilopediaIterator(ruleset) }
+        { ruleset, _ -> BaseUnitType.getCivilopediaIterator(ruleset) }
     ),
     Nation ("Nations",
         CivilopediaImageGetters.nation,
         KeyboardBinding.PediaNations,
         "OtherIcons/Nations",
-        { ruleset, _, _ -> ruleset.nations.values.filter { !it.isSpectator } }
+        { ruleset, _ -> ruleset.nations.values.filter { !it.isSpectator } }
     ),
     Technology ("Technologies",
         CivilopediaImageGetters.technology,
         KeyboardBinding.PediaTechnologies,
         "TechIcons/Philosophy",
-        { ruleset, _, _ -> ruleset.technologies.values }
+        { ruleset, _ -> ruleset.technologies.values }
     ),
     Promotion ("Promotions",
         CivilopediaImageGetters.promotion,
         KeyboardBinding.PediaPromotions,
         "UnitPromotionIcons/Mobility",
-        { ruleset, _, _ -> ruleset.unitPromotions.values }
+        { ruleset, _ -> ruleset.unitPromotions.values }
     ),
     Policy ("Policies",
         CivilopediaImageGetters.policy,
         KeyboardBinding.PediaPolicies,
         "PolicyIcons/Constitution",
-        { ruleset, _, _ -> ruleset.policies.values }
+        { ruleset, _ -> ruleset.policies.values }
     ),
     Belief("Religions and Beliefs",
         CivilopediaImageGetters.belief,
         KeyboardBinding.PediaBeliefs,
         "ReligionIcons/Religion",
-        { ruleset, _, _ -> (
-            ruleset.beliefs.values.asSequence() +
-            BaseBelief.getCivilopediaBeliefsEntry(ruleset) +
-            BaseBelief.getCivilopediaReligionEntry(ruleset)
-        ).toList() }
+        { ruleset, _ -> getCivilopediaReligionEntries(ruleset) }
     ),
     Tutorial ("Tutorials",
         getImage = null,
         KeyboardBinding.PediaTutorials,
         "OtherIcons/ExclamationMark",
-        { _, tutorialController, _ -> tutorialController.getCivilopediaTutorials() }
+        { ruleset, _ -> ruleset.getCivilopediaTutorials() }
```

Comment:
Since the Religion one calls the local `getCivilopediaReligionEntries()`, I think it would be fine to make `getCivilopediaTutorials()` local as well. As in call it with the same signature as the other method: `getCivilopediaTutorials(ruleset)`